.env.example

```diff
@@ -4,14 +4,17 @@ PRIVATE_KEY=
 # RPC 
 RPC_URL=
 
-# Threshold Network
-BLS_KEY=
-
 # Contract upgrades 
 IS_UPGRADE=
 
 # Harhat Contract deployment
 ALCHEMY_KEY=
 
 # Contract verification
-ETHERSCAN_API_KEY=
\ No newline at end of file
+ETHERSCAN_API_KEY=
+
+# Threshold network BLS Point G2 public key x and y coordinates (all uints) used to construct the public key as a G2 point
+BLS_PUBLIC_KEY_X0=
```

Comment:
we should really find a way to convert hex -> PK, though this is fine for now

Reply:
In the solidity, we can convert PK from hex to G2 Point `({x: uint256[2], y: uint256[2]}) `like `BLS.PointG2 memory pk = abi.decode(hex_PK, (BLS.PointG2));`

Reply:
nice, I'm just not 100% sure how to get the right PK hex format from other libs x)

Reply:
Different libs differ in their formatting which is a pain. We can do local tests with the lib we use to generate the keys with signatures before deployment.